Make sure ai_tools can generate the write code for VX4B

LeslieXMOS opened the pull request:
Make sure ai_tools can generate the write code for VX4B

LeslieXMOS committed:
bump version

LeslieXMOS committed:
Change lib_tflite_micro to Leslie's repo to test if whole system working

LeslieXMOS committed:
Update build.sh such that the build with vx4b toolchain

LeslieXMOS committed:
Change lib_tflite_micro submodule to check if whole system workking

LeslieXMOS committed:
Bump lib_tflite_micro version

xalbertoisorna commented:
@LeslieXMOS lib_tflite_micro is merged can go back to upstream, there is a mac issue? not sure what is happenning

LeslieXMOS commented:
> @LeslieXMOS lib_tflite_micro is merged can go back to upstream, there is a mac issue? not sure what is happenning


Let me change lib_tflite micro back to upstream once I'm at airport.

I didnt notice there ia a mac issue, let me check it later.

LeslieXMOS committed:
Change lib_tflite_micro back to xmos version and bump to version to latest

LeslieXMOS commented:
> @LeslieXMOS lib_tflite_micro is merged can go back to upstream, there is a mac issue? not sure what is happenning

lib_tflite_micro is back to upstream.
Mac failure message is useless I think, I will test it with the mac mini in HK office.

LeslieXMOS committed:
Update Bazel cache URL in Jenkinsfile

https://github.com/xmos/ai_tools/pull/974

LeslieXMOS committed:
Update Bazel cache URL and cleanup commands

Updated the Bazel cache URL and added cleanup commands to remove installation files from the temporary directory.

xalbertoisorna commented:
> > @LeslieXMOS lib_tflite_micro is merged can go back to upstream, there is a mac issue? not sure what is happenning
> 
> lib_tflite_micro is back to upstream. Mac failure message is useless I think, I will test it with the mac mini in HK office.

Did you test this locally?

LeslieXMOS commented:
> > > @LeslieXMOS lib_tflite_micro is merged can go back to upstream, there is a mac issue? not sure what is happenning
> > 
> > 
> > lib_tflite_micro is back to upstream. Mac failure message is useless I think, I will test it with the mac mini in HK office.
> 
> Did you test this locally?

No, I can't setup the mac mini to build this

xalbertoisorna committed:
Merge branch 'develop' into feature/vx4b

michaelpoluektov reviewed (approved):
Not tested but merging to develop so we can build a wheel
